pcs/constraint.py

```diff
@@ -122,11 +124,8 @@ def constraint_order_cmd(lib, argv, modifiers):
 
 
 def constraint_show(lib, argv, modifiers):
-    deprecation_warning(
-        "This command is deprecated and will be removed. "
-        "Please use 'pcs constraint config' instead."
-    )
-    return constraint_config_cmd(lib, argv, modifiers)
+    del lib, argv, modifiers
+    raise_command_replaced(["pcs constraint config"], pcs_version="0.11")
```

Comment:
This is actually meant to be deprecated in pcs-0.11. Please, revert your changes.